rules/windows/file/file_event/file_event_win_wermgr_local_privilege_escalation.yml

```diff
@@ -0,0 +1,22 @@
+title: Potential Privilege Escalation Attempt Via Wermgr.Exe.Local
+id: 07a99744-56ac-40d2-97b7-2095967b0e03
+status: experimental
+description: Detects potential privilege escalation attempt via the creation of the "Wermgr.Exe.Local" folder inside the "System32" directory in order to sideload "comctl32.dll"
+references:
+    - https://github.com/binderlabs/DirCreate2System
+author: Nasreddine Bencherchali
+date: 2022/12/16
+tags:
+    - attack.defense_evasion
+    - attack.persistence
+    - attack.privilege_escalation
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection:
+        TargetFilename|startswith: 'C:\windows\system32\wermgr.exe.local\'
```

Comment:
No `TargetFilename|endswith: '\comctl32.dll'`
```xml
<EventData>
  <Data Name="RuleName">DLL</Data> 
  <Data Name="UtcTime">2022-12-16 18:33:30.501</Data> 
  <Data Name="ProcessGuid">{8b8a3bea-b9f7-639c-4301-00000000ae00}</Data> 
  <Data Name="ProcessId">6888</Data> 
  <Data Name="Image">C:\temp\DirCreate2System-main\bin\dircreate2system.exe</Data> 
  <Data Name="TargetFilename">C:\Windows\System32\wermgr.exe.local\amd64_microsoft.windows.common-controls_6595b64144ccf1df_6.0.19041.1110_none_60b5254171f9507e\comctl32.dll</Data> 
  <Data Name="CreationUtcTime">2022-12-16 18:33:30.501</Data> 
  <Data Name="User">SIMU\Administrateur</Data> 
  </EventData>
```

Reply:
You're right. I missed it. Will add it.